lms/static/js/dashboard/legacy.js

```diff
@@ -37,8 +37,6 @@
             notifications.focus();
         }
 
-        $('.action-more').bind('click', toggleCourseActionsDropdown);
```

Comment:
All this was moved to the ```course_view``` backbone view with same names. if I let it stay in this file some of the events misbehave